datasetIngestor/ingestDataset.go

```diff
@@ -114,7 +114,8 @@ func createDataset(client *http.Client, APIServer string, metaDataMap map[string
 func getEndpoint(dstype string) (string, error) {
 	switch dstype {
 	case "raw":
-		return "/RawDatasets", nil
+		// return "/RawDatasets", nil
+		return "/Datasets", nil
```

Comment:
This is a v3 to v4 difference. V3 had separate endpoints for all classes, whereas V4 just has the main `/dataset` endpoint.

There is a `/Datasets` in v3. Maybe that could be used everywhere and still be backwards compatible?